de/developer/api/methods.md

```diff
@@ -2,7 +2,7 @@ Deinen ersten Eintrag hinzufügen
 --------------------------------
 
 Dokumentation für diese Methode:
-<http://localhost:8000/api/doc#post--api-entries>.{\_format}
+https://app.wallabag.it/api/doc#post--api-entries.{_format}
```

Comment:
the underscores, don't they need to be escaped anymore? I remember the online docs made the word "format" underlined without the escaping

Reply:
I don't think.